cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -134,8 +136,11 @@ private void checkKafkaTopic(Set<String> problemKeys) {
     if (!isValidKafkaTopic(Constants.Logging.KAFKA_TOPIC)) {
       problemKeys.add(Constants.Logging.KAFKA_TOPIC);
     }
-    if (!isValidKafkaTopic(Constants.Audit.KAFKA_TOPIC)) {
-      problemKeys.add(Constants.Audit.KAFKA_TOPIC);
+  }
+
+  private void checkMessagingTopics(Set<String> problemKeys) {
```

Comment:
Do we need to check if all of the system topics (from the comma separated list from cConf) to be created are all valid IDs?